operator/pkg/reconciliation/secrets_test.go

```diff
@@ -84,17 +84,6 @@ func Test_validateSuperuserSecretContent(t *testing.T) {
 		valid           bool
 		message         string
 	}{
-		{
-			secretNil: true,
-			valid:     true,
-			message:   "validation should pass when a secret is going to be auto-generated",
-		},
-		{
-			superuserSecret: "my-fun-secret",
-			secretNil:       true,
-			valid:           false,
-			message:         "validation should fail when the user explicitly specifies a secret but it does not exist",
-		},
```

Comment:
Checks for these were moved outside `validateSuperuserSecretContent()` so removing these cases here.